app/scripts/lib/createMetaRPCHandler.js

```diff
@@ -20,6 +60,19 @@ const createMetaRPCHandler = (api, outStream) => {
     let error;
     try {
       result = await api[data.method](...data.params);
+
```

Comment:
There are a lot of actions besides 'trackMetaMetricsEvent', 'getState' which do no change state. But yep will be hard to mention all of those here.

Reply:
yeah... we should probably try to get a comprehensive list here

Reply:
I'd recommend making an object with keys of method names and checking the object for key existing, rather than an includes call, for performance reasons.

Reply:
I would suggest just use `getState`. Creating a method list here can cause error if a method is modified but this list is not updated.

Having an extra call to persist state is not a huge overhead I think.